### ISSUE TITLE
Super Mario Bros. Wonder: Final Cutscene broken

### ISSUE THREAD

**Issue by ilonachan** [yuzu-emu/yuzu#12788]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1692 / Early Access 4084

### Description of Issue

After clearing the final level of the main game, "Bowser's Rage Stage", a bunch of cutscenes follow: The first with the royal seeds plays fine, so does the one of Castle Bowser disappearing. After that a small single-screen freewalk (iirc) cutscene with poplins should play, and the credits minigame should start. These last two consistently don't trigger, instead a blank gray screen is shown while the background music of the poplin cutscene can be heard in the background. The end credits never start. It seems to be possible to perform menuing in this state, but the menu isn't visible either.

### Expected Behavior

The poplin cutscene should be shown on screen in sync with the audio, then afterwards the end credits minigame should start.

### Reproduction Steps

1. Beat "Bowser's Rage Stage" ~~for the first time~~
2. After the castle disappearing scene, the game is softlocked with a blank screen.

This bug is particularly terrible to reproduce, because it requires playing through the entire level each time to test. Here's a save file that hopefully makes attempts easy, and also has the Bowser dialogue marked seen so it can be skipped.
[pre_clear.zip](https://github.com/yuzu-emu/yuzu/files/14054083/pre_clear.zip)
But since the end credits happen anytime that stage is beaten, a 100% save (like [this one](https://gamebanana.com/mods/474758)) can also reproduce it.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/14054078/yuzu_log.txt)
To summarize, starting from the castle disappearing cutscene, the following line is repeated:
`HW.GPU <Warning> ./video_core/buffer_cache/buffer_cache.h:StorageBufferBinding:1784: Failed to find storage buffer for cbuf index 0`

### System Configuration

CPU: AMD Ryzen 7 4700G (with builtin Radeon Graphics)
GPU/Driver: NVIDIA RTX 2060 Super, proprietary driver
RAM: 15.00 GiB (I thought it was 16 GiB but Yuzu logs claim it's just 15)
OS: Manjaro Linux

**Comment by Squall-Leonhart** [comment]
>  HW.GPU <Warning> ./video_core/buffer_cache/buffer_cache.h:StorageBufferBinding:1784: Failed to find storage buffer for cbuf index 0

This is benign and happens all throughout the game.

You are running without the Stabilise fps mod, which is reported to fix issues like this.

**Comment by ilonachan** [comment]
Oh nice, that actually fixed it! For anyone reading this later, [here's where I found the mod](https://github.com/theboy181/switch-ptchtxt-mods/tree/main/Super%20Mario%20Bros.%20Wonder/%5B010015100B514000%5D/v1.0.0) (just linking it bc google/duck search didn't give a great result immediately). I'm still gonna leave this open, because I feel like this should eventually be correctly emulated without workarounds, but the workaround does work.
